lib/openshift/resource.rb

```diff
@@ -49,6 +49,14 @@ def exists?(user: nil, result: {}, quiet: false)
       elsif result[:response] =~ /Unable to connect to the server/
         logger.info(result[:response])
         return false
+      # Adding below because when in Single node cluster node restarted then SNO not have other nodes to communicate which cause connection timeout issue.
+      # And to prevent premature exit as describe in OCPQE-8043.
+      no_of_master = env.nodes.length
```

Comment:
Rename the variable `no_of_master` to `num_of_master`

Reply:
The variable is assigned length of nodes, so maybe `num_of_nodes` is more appropriate var name here.